R/wavelets.R

```diff
@@ -113,10 +115,14 @@ Morlet <- R6::R6Class(
 #'
 #' @param m the order of the derivative
 #'
+#' @importFrom calculus hermite
```

Comment:
You don't need the `importFrom` here if you are calling `calculus::termite()` anyway - with the `calculus::`.

Reply:
The usage of 

```
h <- calculus::hermite(m)[[m + 1]]$f
eval(parse(text = paste('He_n <- function(x) {', h, '}', sep = '')))
```

is quite dangerous. I'm assuming there's no other way to obtain that polynomial.

Reply:
I didn't find a better one, let's put it like that ...

Reply:
I added the `importFrom` because of a - I don't remember exactly - warning? in `devtools::check()` ...
Let me remove it and see what happens exactly, maybe something different is needed ...

Reply:
hm strange, I don't get that anymore ... :-)